contracts/Minter.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.5.0) (token/ERC20/presets/ERC20PresetMinterPauser.sol)
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @dev {ERC20} token, including:
+ *
+ *  - a minter role that allows for token minting (creation)
+ */
+contract Minter is ERC20, Ownable {
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
```

Comment:
Are these roles used? If not, reove them and clean functions documentation.

Reply:
Replaced with authorizedAddresses